model/datasetLandingPageCensus/model.go

```diff
@@ -10,15 +10,33 @@ type Page struct {
 	model.Page
 	DatasetLandingPage DatasetLandingPage            `json:"data"`
 	ContactDetails     contactDetails.ContactDetails `json:"contact_details"`
+	HasContactDetails  bool                          `json:"has_contact_details"`
 }
 
 // DatasetLandingPage contains properties related to the census dataset landing page
 type DatasetLandingPage struct {
+	GuideContents GuideContents
 	Sections      map[string]Section
 	ShareDetails  ShareDetails
 	Methodologies []Methodology `json:"methodology"`
 }
 
+// GuideContents contains the contents of the page and the language attribute
+type GuideContents struct {
+	GuideContent []Content `json:"guide_content"`
+	Language     string    `json:"language"`
+}
+
+/* Content maps the content details.
+The visible text can be either a 'Title' or a 'LocaliseKey'.
```

Comment:
A small thing, but it may be worth highlighting in the comment here that the `LocaliseKey` has to correspond to a localisation key found in localisation files found in `assets/locales`, otherwise the page will error